indico/modules/rb_new/operations/rooms.py

```diff
@@ -73,6 +75,29 @@ def _query_managed_rooms(user):
     return Room.query.filter(Room.is_active, db.or_(Room.owner == user, is_manager))
 
 
+# TODO
+@no_autoflush
+def _populate_room(room, properties):
+    basic_props = [prop for prop in room_args.keys() if prop not in ['available_equipment', 'unbookable_periods',
```

Comment:
calling `.keys()` here isn't needed; iterating a dict always gives you the keys

Reply:
also, i don't like the idea of using `room_args` here very much - why not just iterate over `properties`?